styles/sass/main.scss

```diff
@@ -13,6 +13,23 @@
   --black: #000;
 }
 
+.grid {
+  display: grid;
+  grid-template-columns: repeat(2, calc(50% - 24px));
+  grid-gap: 48px;
+
+  @include breakpoint(tablet) {
+    grid-template-columns: 1fr;
+    width: auto;
+    height: auto;
+  }
+}
+
+.small-body-decoration {
```

Comment:
you might use `@extend .small-body;` here. 
Then there will be no need to wrap a `p` inside a `div` to get the same result 

(you might need to add `padding-inline-start: 24px` in that case)
It's up to you ;)

Reply:
Done